### DISCUSSION TITLE
2022 model (Legion 5 15ARH7H)

### DISCUSSION THREAD

**Discussion by sterlingphoenix** [4JX/L5P-Keyboard-RGB#84] [Ideas]
Hi there,

I've downloaded and added this model to the source, but... I can't make PRs against this project (probably I'm doing something wrong). It's a trivial addition -- the USB device ID is `048d:c975` (:

**Comment by 4JX** [comment]
Can you test [this version](https://github.com/4JX/L5P-Keyboard-RGB/actions/runs/3100274787) to confirm it works?

**Reply by sterlingphoenix** [reply]
I'm not sure how to download that version (anything I click takes me to the main release) but it looks pretty much exactly like the version I made (except that I put the 2022 version before the 2021 version to be consistent).

**Reply by 4JX** [reply]
The artifacts section at the bottom contains Windows and Linux binaries

**Reply by sterlingphoenix** [reply]
Ok, works perfectly!
